client/js/components/day/employee.js

```diff
@@ -22,4 +26,49 @@ var Employee = React.createClass({
 
 });
 
-module.exports = Employee;
\ No newline at end of file
+// see docs http://gaearon.github.io/react-dnd/docs-drag-source.html
+
+var type = 'employee';
+
+var spec = {
+
+  beginDrag: function(props, monitor, component){
+    // return a plain JavaScript object describing the data being dragged
+    return {
+      name: props.name,
+      rating: props.rating,
+      gigID: props.gigID,
+      employeeID: props.employeeID
+    };
+  },
+
+  endDrag: function(props, monitor, component){
+    if (monitor.didDrop()){
+       // the drop target specified a drop result by
+       // returning a plain object from its drop() method,
+       // it will be available as monitor.getDropResult()
+      var toGig = monitor.getDropResult();
+      ViewActionCreator.moveStaff({
+        fromGig: props.gigID,
+        fromGroup: props.group,
+        toGig: toGig.gigID,
+        toGroup: toGig.group,
+        employeeID: props.employeeID
+      });
+    }
+  }
+
+  // can implement this once we have approval flow. will allow us
+  // to prevent dragging of approved contractors
+
+    // canDrag: function (props) {
```

Comment:
It would be great to uncomment this canDrag function and just return true for now. Once authentication is working we can swap that out for props.isReady. The advantage being that we can test to make sure this  drag authentication functionality is workng right now.